src/preload.ts

```diff
@@ -0,0 +1,7 @@
+// All of the Node.js APIs are available in the preload process.
+// It has the same sandbox as a Chrome extension.
+window.addEventListener("DOMContentLoaded", () => {
+  for (const versionType of ["chrome", "electron", "node"]) {
+    document.getElementById(`${versionType}-version`).innerText = (process.versions as any)[versionType];
```

Comment:
I'm not the best at TypeScript, so if there's a better way to deal with `process.versions`, suggestions are much appreciated.

Reply:
i'd recommend following this: https://github.com/electron/electron-quick-start/blob/master/preload.js

Reply:
It was initially a copy/paste :sweat_smile: 

Copying as-is generates the following TypeScript error:

```
src/preload.ts:5:67 - error TS7053: Element implicitly has an 'any' type because expression of type '
string' can't be used to index type 'ProcessVersions'.
  No index signature with a parameter of type 'string' was found on type 'ProcessVersions'.

5     document.getElementById(`${versionType}-version`).innerText = process.versions[versionType];
                                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

Reply:
oh yes sorry i mean the ts-ified version of that :)
```
window.addEventListener('DOMContentLoaded', () => {
  const replaceText = (selector, text) => {
    const element = document.getElementById(selector)
    if (element) element.innerText = text
  } 
  
  for (const type of ['chrome', 'node', 'electron']) {
    replaceText(`${type}-version`, (process.versions as any)[type])
  }
})
```

this one just ensures no errors occur if an element isn't found for some weird load time reasons, which has occurred rarely

Reply:
Ah, I see. I had copy/pasted the old version. Updated in https://github.com/electron/electron-quick-start-typescript/pull/26/commits/1fff71bfdcac51a10d0bd6a04b24869196339f38